addons/web_editor/static/lib/html2canvas.js

```diff
@@ -5603,15 +5603,19 @@
               return reject("No window assigned for iframe");
           }
           var documentClone = cloneWindow.document;
-          cloneWindow.onload = iframe.onload = function () {
-              cloneWindow.onload = iframe.onload = null;
-              var interval = setInterval(function () {
-                  if (documentClone.body.childNodes.length > 0 && documentClone.readyState === 'complete') {
-                      clearInterval(interval);
-                      resolve(iframe);
-                  }
-              }, 50);
-          };
+          /*
+            Patch for safari v 15.4+ in ios: onload is never called.
```

Comment:
Could you please change the comment to be easy to grep when the lib will be updated ?
Thanks.

```suggestion
            ODOO Patch for safari v 15.4+ in ios: onload is never called.
```